packages/core/src/utils/order.ts

```diff
@@ -0,0 +1,13 @@
+export const orderObject = <T extends { [key: string]: any }>(obj: T): T => {
+  const newObj = {} as T;
+  for (const key of Object.keys(obj).sort()) {
```

Comment:
I think it would be more accurate to handle JSON arrays differently, because their order is already strict (two JSON arrays with a different order are expected to be different, unlike object keys).